docs/en-US/component/skeleton.md

```diff
@@ -77,7 +77,7 @@ skeleton/rendering-with-data
 
 ## Avoiding rendering bouncing.
 
-Sometimes API responds very quickly, when that happens, the skeleton just gets rendered to the DOM then it needs to switch back to real DOM, that causes the sudden flashy. To avoid such thing, you can use the `throttle` attribute.
+Sometimes API responds very quickly, when that happens, the skeleton just gets rendered to the DOM then it needs to switch back to real DOM, that causes the sudden flashy. To avoid such thing, you can use the `throttle` attribute. The `throttle` attribute supports two values: `number` and `object`. When passing a `number`, it is equivalent to `{leading: xxx}`, controlling the throttling of the skeleton screen display. Of course, you can also control the throttling of the skeleton screen disappearance by passing `{trailing: xxx}`. When the initial value of loading is true, you can set `{initVal: true}` to control the immediate display of the initial skeleton screen without throttling.
```

Comment:
The content here may be created as a separate subheading, and the supported version number can be conveniently marked.

Reply:
ok